app/components/gcse_preview_component.rb

```diff
@@ -0,0 +1,55 @@
+class GcsePreviewComponent < ViewComponent::Base
```

Comment:
Apart from the class name is is every else a duplication?

Is there some ways of reuse like `helpers` or inheritance? (Not too fuss if its clearer this way.)

Reply:
Currently doing a little refactor to DRY this out will push up in a sec 👍